tests/Application/src/Entity/Author.php

```diff
@@ -13,38 +13,25 @@
 
 namespace App\Entity;
 
+use App\Repository\AuthorRepository;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
-use JMS\Serializer\Annotation as Serializer;
-use Sylius\Component\Resource\Model\ResourceInterface;
+use Doctrine\ORM\Mapping\Entity;
 
-/**
- * @Serializer\ExclusionPolicy("all")
- */
 #[ORM\MappedSuperclass]
 #[ORM\Table(name: 'app_author')]
-class Author implements ResourceInterface
+#[Entity(repositoryClass: AuthorRepository::class)]
```

Comment:
```suggestion
#[ORM\Entity(repositoryClass: AuthorRepository::class)]
```
The `Mapping` import is already in place, and it's already like that in the other entities 💃

Reply:
done, you are right